src/SingleReview.jsx

```diff
@@ -0,0 +1,94 @@
+import {useParams} from 'react-router-dom'
+import {useState,useEffect} from 'react'
+import { getSingleReview } from './api'
+import { formatDate } from './api'
+import AddReviewLikes from './AddReviewLikes'
+
+function SingleReview() {
+    const [singleReview,setSingleReview] = useState({})
+    const {review_id} = useParams();
+    
+
+    const [isLoading, setIsLoading] = useState(false)
+
+    const [finishedLoading, setFinishedLoading] = useState(false)
+    useEffect(() => {
+        setIsLoading(true)
+        setFinishedLoading(false)
+        getSingleReview(review_id).then((review) => {
+            setSingleReview(review)
+            setFinishedLoading(true)
+            const animation = document.querySelector('.rocket')
```

Comment:
The animation is super cool!! I'm not sure about mixing DOM and React together though, it's generaly deemed as not best practice. As its a portfolio piece If you get time at the end try to have a look at react animations or remove it should you need as It may not look the best if an employer was to look though your code.